metricbeat/module/docker/healthcheck/data.go

```diff
@@ -0,0 +1,52 @@
+package healthcheck
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/docker"
+
+	dc "github.com/fsouza/go-dockerclient"
+	"reflect"
+	"strings"
+)
+
+func eventsMapping(containersList []dc.APIContainers, m *MetricSet) []common.MapStr {
+	myEvents := []common.MapStr{}
+	for _, container := range containersList {
+		returnevent := eventMapping(&container, m)
+		// Compare event to empty event
+		if !reflect.ValueOf(returnevent).IsNil() {
```

Comment:
You should be able to just do `returnevent != nil` here instead of using reflect package.